discovery/kubernetes/node.go

```diff
@@ -15,16 +15,15 @@ package kubernetes
 
 import (
 	"context"
-	"net"
-	"strconv"
-
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	"github.com/pkg/errors"
 	"github.com/prometheus/common/model"
 	apiv1 "k8s.io/api/core/v1"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
+	"net"
+	"strconv"
```

Comment:
It seems these importes were moved by accident.

Reply:
Oh, It should have been modified accidentally, I have rolled it back.